03-attribute-operations.Rmd

```diff
@@ -275,15 +255,15 @@ world_continents = world %>%
 world_continents
 ```
 
-`sf` objects are well-integrated with the **tidyverse**, as illustrated by the fact that the aggregated objects preserve the geometry of the original `world` object.
+`sf` objects are well-integrated with the **tidyverse**, as illustrated by the fact that the aggregated objects preserve the geometry of the original `world` object.^[Such a spatial aggregation of polygon data is know as "to dissolve polygons" in the GIS world.]
```

Comment:
Yes that's a good plan - it's important to note that it does a *spatial* data operation 'under the hood' which is clever.